src/Network/Network/help/New-AzNetworkWatcherNetworkConfigurationDiagnosticProfile.md

```diff
@@ -0,0 +1,229 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version:
```

Comment:
@irrogozh make sure to add the online help URL here

`https://docs.microsoft.com/en-us/powershell/module/az.network/new-aznetworkwatchernetworkconfigurationdiagnosticprofile`

Reply:
Done